libs/blocks/caas-marquee/caas-marquee.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I just saw we had the same thought to get things fixed. - https://github.com/adobecom/milo/pull/2025 
we probably shouldn't have this empty line no? 
Maybe we close this PR and just merge mine?